Samples/src/PhaseEstimation/BayesianPhaseEstimation.qs

```diff
@@ -241,40 +229,38 @@ namespace Microsoft.Quantum.Samples.PhaseEstimation {
     /// # Output
     /// An approximation of ∫_I f(x) dx, where I is the interval [x₀, xₘ],
     /// and where m is the length of `xs`.
-    function Integrate (xs : Double[], ys : Double[]) : Double {
-        
+    function Integrated(xs : Double[], ys : Double[]) : Double {
         mutable sum = 0.0;
-        
+
         for (idxPoint in 0 .. Length(xs) - 2) {
             let trapezoidalHeight = (ys[idxPoint + 1] + ys[idxPoint]) * 0.5;
             let trapezoidalBase = xs[idxPoint + 1] - xs[idxPoint];
             set sum += trapezoidalBase * trapezoidalHeight;
         }
-        
+
         return sum;
     }
-    
-    
+
+
     /// # Summary
     /// Given two arrays, returns a new array that is the pointwise product
     /// of each of the given arrays.
-    function MultiplyPointwise (left : Double[], right : Double[]) : Double[] {
-        
+    function PointwiseProduct(left : Double[], right : Double[]) : Double[] {
         mutable product = new Double[Length(left)];
-        
+
         for (idxElement in 0 .. Length(left) - 1) {
```

Comment:
Possible use of `IndexRange` library function.

Reply:
Good point! Fixed.